### DISCUSSION TITLE
Number does not save

### DISCUSSION THREAD

**Discussion by MauriceHandwerker** [project-error/npwd#666] [Q&A]
Hi I currently have a problem with the phone.
I Installed it and everything. 
But for some reason the number does not save in the SQL Database

**Comment by duskimm** [comment]
Hello @WilderMO I found the problem you are probably using the esx_multicharacter ( like me ) and it puts a prefix in front of the identifier like char1: ( identifier ) , the solution is to not use the esx_multicharacter and stop it in the es_extended config (Config.Multichar = false -- Enable support for esx_multicharacter)

@jfrader @dunak-debug @itschip @Mojito-Fivem @TasoOneAsia
